### PR TITLE
Fx misc issues in gui modules.

### PR DESCRIPTION
**Pull request by marcofavorito**
This PR aims to fix some issues related to the GUI modules. The changes have been commented inline.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -14,8 +14,6 @@ docker = "*"
 flake8-docstrings = "*"
 
 [packages]
-flask-sqlalchemy = "*"
-sqlalchemy = "*"
```
Comment (marcofavorito) [inline Pipfile]: The old GUI is not supported anymore.

File: tac/gui/dashboards/agent.py [hunk 2]
```diff
@@ -83,7 +83,6 @@ def __init__(self, agent_name: str,
 
     def init(self):
         """Re-initiate the AgntDashboard."""
```
Comment (DavidMinarsch) [inline tac/gui/dashboards/agent.py]: can you fix the typo in the docstring? > AgentDashboard

File: tac/gui/dashboards/agent.py [hunk 3]
```diff
@@ -83,7 +83,6 @@ def __init__(self, agent_name: str,
...
-        self.viz.delete_env(self.env_name)
```
Comment (marcofavorito) [inline tac/gui/dashboards/agent.py]: This line caused some problems on the Visdom server - we were trying to delete a non-existing environment.

File: tac/gui/dashboards/base.py [hunk 4]
```diff
@@ -42,7 +42,7 @@ def stop(self):
 
 def start_visdom_server() -> subprocess.Popen:
     """Start the visdom server."""
-    visdom_server_args = ["python", "-m", "visdom.server", "-env_path", os.path.join(CUR_DIR, ".visdom_env")]
+    visdom_server_args = ["python", "-m", "visdom.server", "-env_path", os.path.join(CUR_DIR, "..", ".visdom_env")]
```
Comment (marcofavorito) [inline tac/gui/dashboards/base.py]: Update the relative path to `.visdom_env` to comply with the new package structure.

File: tac/gui/dashboards/controller.py [hunk 5]
```diff
@@ -66,7 +66,7 @@ def from_datadir(datadir: str, env_name: str) -> 'ControllerDashboard':
         game_data = json.load(open(game_data_json_filepath))
         game = Game.from_dict(game_data)
         game_stats = GameStats(game)
-        return ControllerDashboard(game_stats, env_name)
+        return ControllerDashboard(game_stats, env_name=env_name)
```
Comment (marcofavorito) [inline tac/gui/dashboards/controller.py]: forward the `env_name` argument properly.

File: tac/gui/dashboards/agent.py (hunk 2 above)
Comment (marcofavorito) [reply tac/gui/dashboards/agent.py]: yes

**Pull request merged**